### PR TITLE
display publisher prefLabel if present

### PR DESCRIPTION
**Pull request by oyvindis**
Jira issue link: [link](LINK_HERE)

> check applicable boxes

- [ ] I have made tests to match the user stories
- [ ] This code does not require tests that match user stories

### CODE REVIEW

File: applications/search/src/lib/stringUtils.js [hunk 1]
```diff
@@ -15,3 +15,15 @@ export function addValue(list, value) {
   ret.push(value);
   return ret.join(',');
 }
+
```
Comment (markkopaas) [inline applications/search/src/lib/stringUtils.js]: Vi kan jo bruke den i stedet?
https://lodash.com/docs/4.17.10#capitalize

File: applications/search/src/pages/search-page/results-concepts/results-concepts.component.jsx [hunk 2]
```diff
@@ -38,15 +41,17 @@ export class ResultsConcepts extends React.Component {
...
     const children = items =>
       items.map((item, index) => {
-        let creator;
-        if (item.creator && item.creator.name) {
-          creator = item.creator.name;
-        }
+        const { creator } = item;
+
+        const publisherPrefLabel =
+          getTranslateText(_get(creator, ['prefLabel'], null)) ||
```
Comment (markkopaas) [inline applications/search/src/pages/search-page/results-concepts/results-concepts.component.jsx]: _.get har default undefined, som er god nåk.

File: applications/search/src/pages/search-page/results-dataset/search-hit-item/search-hit-item.component.jsx [hunk 3]
```diff
@@ -56,19 +58,18 @@ const renderFormats = (source, code, distributionTypeItems) => {
 
 const renderPublisher = source => {
   const { publisher } = source;
-  if (publisher && publisher.name) {
+  const publisherPrefLabel =
+    getTranslateText(_get(publisher, ['prefLabel'], null)) ||
```
Comment (markkopaas) [inline applications/search/src/pages/search-page/results-dataset/search-hit-item/search-hit-item.component.jsx]: igjen, bruk _.get default

File: applications/search/src/pages/search-page/filter-option/filter-option.component.jsx [hunk 4]
```diff
@@ -15,7 +17,7 @@ export const FilterOption = props => {
     displayClass
   } = props;
 
-  const optionLabel = `${label.charAt(0)}${label.substring(1).toLowerCase()}`;
+  const optionLabel = labelRaw || `${capitalizeFirstLetter(label)}`;
```
Comment (markkopaas) [inline applications/search/src/pages/search-page/filter-option/filter-option.component.jsx]: capitalizeFirstLetter(label) er nåk

File: applications/search/src/lib/stringUtils.js (hunk 1 above)
Comment (oyvindis) [reply applications/search/src/lib/stringUtils.js]: Replaced

**Pull request merged**